pontoon/batch/utils.py

```diff
@@ -30,6 +40,31 @@ def get_translations_info(translations, locale):
     return count, translated_resources, changed_entities
 
 
+def ftl_find_and_replace(string, find, replace):
+    """Replace text values in an FTL string.
+
+    :arg string string: a serialized FTL string
+    :arg string find: a string to search for, and replace, in translations
+    :arg string replace: what to replace the original string with
+
+    :returns: a serialized FTL string
+    """
+
+    def fun(value):
```

Comment:
Perhaps rename `fun` to something more descriptive, e.g. `replace_text_elements`?